pkg/testkit/mocksessionmanager.go

```diff
@@ -157,11 +157,13 @@ func (msm *MockSessionManager) GetInternalSessionStartTSList() []uint64 {
 	ret := make([]uint64, 0, len(msm.internalSessions))
 	for internalSess := range msm.internalSessions {
 		se := internalSess.(sessionctx.Context)
-		sessVars := se.GetSessionVars()
-		sessVars.TxnCtxMu.Lock()
-		startTS := sessVars.TxnCtx.StartTS
-		sessVars.TxnCtxMu.Unlock()
-		ret = append(ret, startTS)
```

Comment:
The original implementation is not correct. The `sessVars.TxnCtx.StartTS` is not cleared after `Commit`, so that a pending session will always have a small start ts.